build-ami.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
-set -ex
+set -exu
 
-APPLIANCE_IMAGE=${1-registry.tld/conjur-appliance}
-APPLIANCE_IMAGE_TAG=${2-latest}
+IMAGE="$1"  # ex: registry.tld/conjur-appliance:4.9-stable
+TAG="${IMAGE##*:}"
 
 if [ ! -f conjur-appliance.tar.gz ]; then
-  docker pull ${APPLIANCE_IMAGE}:${APPLIANCE_IMAGE_TAG}
-  docker save ${APPLIANCE_IMAGE}:${APPLIANCE_IMAGE_TAG} > conjur-appliance.tar
+  docker pull $IMAGE
+  docker save $IMAGE > conjur-appliance.tar
```

Comment:
Can you create a followup task to avoid this tar -> scp step? It takes a long time. Instead, we should just launch these instances so they are able to pull the image from a registry. We can wipe the creds after pull